<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Galal" failures="4" tests="7" name="tests.ui.SecurityTest" time="326.708" errors="0" timestamp="2025-03-25T01:20:51 EET" skipped="0">
  <testcase classname="tests.ui.SecurityTest" name="verifySessionHandling" time="34.469">
    <system-out>
      <![CDATA[🟢 Test Started: verifySessionHandling (🔢 Test Case ID: TC-021)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

✅ Test PASSED: verifySessionHandling (Duration: 17272ms)
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-021\TC-021_screenshot.png' height='300' width='400'/><br>]]>
    </system-out>
  </testcase> <!-- verifySessionHandling -->
  <testcase classname="tests.ui.SecurityTest" name="verifyHTTPSProtocol" time="36.721">
    <system-out>
      <![CDATA[🟢 Test Started: verifyHTTPSProtocol (🔢 Test Case ID: TC-292)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

✅ Test PASSED: verifyHTTPSProtocol (Duration: 19358ms)
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-292\TC-292_screenshot.png' height='300' width='400'/><br>]]>
    </system-out>
  </testcase> <!-- verifyHTTPSProtocol -->
  <testcase classname="tests.ui.SecurityTest" name="verifyProtectionAgainstXSS" time="18.530">
    <failure message="XSS attack successful: payload is reflected in the Email field. expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: XSS attack successful: payload is reflected in the Email field. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertFalse(Assert.java:79)
at tests.ui.SecurityTest.verifyProtectionAgainstXSS(SecurityTest.java:277)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:102)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:91)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
    </failure>
    <system-out>
      <![CDATA[🟢 Test Started: verifyProtectionAgainstXSS (🔢 Test Case ID: TC-126)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

❌ Test FAILED: verifyProtectionAgainstXSS - XSS attack successful: payload is reflected in the Email field. expected [false] but found [true]
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-126\TC-126_screenshot.png' height='300' width='400'/><br>]]>
    </system-out>
  </testcase> <!-- verifyProtectionAgainstXSS -->
  <testcase classname="tests.ui.SecurityTest" name="verifyPaymentGatewaySecurity" time="49.372">
    <failure message="Card number is stored in plaintext. It should be encrypted or masked. expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Card number is stored in plaintext. It should be encrypted or masked. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertFalse(Assert.java:79)
at tests.ui.SecurityTest.verifyPaymentGatewaySecurity(SecurityTest.java:186)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:102)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:91)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
    </failure>
    <system-out>
      <![CDATA[🟢 Test Started: verifyPaymentGatewaySecurity (🔢 Test Case ID: TC-091)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

❌ Test FAILED: verifyPaymentGatewaySecurity - Card number is stored in plaintext. It should be encrypted or masked. expected [false] but found [true]
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-091\TC-091_screenshot.png' height='300' width='400'/><br>]]>
    </system-out>
  </testcase> <!-- verifyPaymentGatewaySecurity -->
  <testcase classname="tests.ui.SecurityTest" name="verifyPasswordStrengthRequirements" time="24.509">
    <failure message="Error message did not include &#039;at least 8 characters&#039; requirement. expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Error message did not include 'at least 8 characters' requirement. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at tests.ui.SecurityTest.verifyPasswordStrengthRequirements(SecurityTest.java:149)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:102)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:91)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
    </failure>
    <system-out>
      <![CDATA[🟢 Test Started: verifyPasswordStrengthRequirements (🔢 Test Case ID: TC-034)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

❌ Test FAILED: verifyPasswordStrengthRequirements - Error message did not include 'at least 8 characters' requirement. expected [true] but found [false]
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-034\TC-034_screenshot.png' height='300' width='400'/><br>]]>
    </system-out>
  </testcase> <!-- verifyPasswordStrengthRequirements -->
  <testcase classname="tests.ui.SecurityTest" name="verifyChangePasswordFromDashboard" time="54.418">
    <system-out>
      <![CDATA[🟢 Test Started: verifyChangePasswordFromDashboard (🔢 Test Case ID: TC-118)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

✅ Test PASSED: verifyChangePasswordFromDashboard (Duration: 35755ms)
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-118\TC-118_screenshot.png' height='300' width='400'/><br>]]>
    </system-out>
  </testcase> <!-- verifyChangePasswordFromDashboard -->
  <testcase classname="tests.ui.SecurityTest" name="verifyAccountLockoutAfterMultipleFailedLoginAttempts" time="108.689">
    <failure message="Account was not locked after multiple failed login attempts. expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Account was not locked after multiple failed login attempts. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at tests.ui.SecurityTest.verifyAccountLockoutAfterMultipleFailedLoginAttempts(SecurityTest.java:102)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:102)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:91)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
    </failure>
    <system-out>
      <![CDATA[🟢 Test Started: verifyAccountLockoutAfterMultipleFailedLoginAttempts (🔢 Test Case ID: TC-027)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

❌ Test FAILED: verifyAccountLockoutAfterMultipleFailedLoginAttempts - Account was not locked after multiple failed login attempts. expected [true] but found [false]
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-027\TC-027_screenshot.png' height='300' width='400'/><br>]]>
    </system-out>
  </testcase> <!-- verifyAccountLockoutAfterMultipleFailedLoginAttempts -->
  <system-out>
    <![CDATA[🚀 Starting Test Suite: Surefire test

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.89

🟢 Test Started: verifyAccountLockoutAfterMultipleFailedLoginAttempts (🔢 Test Case ID: TC-027)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

❌ Test FAILED: verifyAccountLockoutAfterMultipleFailedLoginAttempts - Account was not locked after multiple failed login attempts. expected [true] but found [false]
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-027\TC-027_screenshot.png' height='300' width='400'/><br>
🟢 Test Started: verifyChangePasswordFromDashboard (🔢 Test Case ID: TC-118)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

✅ Test PASSED: verifyChangePasswordFromDashboard (Duration: 35755ms)
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-118\TC-118_screenshot.png' height='300' width='400'/><br>
🟢 Test Started: verifyHTTPSProtocol (🔢 Test Case ID: TC-292)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

✅ Test PASSED: verifyHTTPSProtocol (Duration: 19358ms)
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-292\TC-292_screenshot.png' height='300' width='400'/><br>
🟢 Test Started: verifyPasswordStrengthRequirements (🔢 Test Case ID: TC-034)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

❌ Test FAILED: verifyPasswordStrengthRequirements - Error message did not include 'at least 8 characters' requirement. expected [true] but found [false]
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-034\TC-034_screenshot.png' height='300' width='400'/><br>
🟢 Test Started: verifyPaymentGatewaySecurity (🔢 Test Case ID: TC-091)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

❌ Test FAILED: verifyPaymentGatewaySecurity - Card number is stored in plaintext. It should be encrypted or masked. expected [false] but found [true]
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-091\TC-091_screenshot.png' height='300' width='400'/><br>
🟢 Test Started: verifyProtectionAgainstXSS (🔢 Test Case ID: TC-126)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

❌ Test FAILED: verifyProtectionAgainstXSS - XSS attack successful: payload is reflected in the Email field. expected [false] but found [true]
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-126\TC-126_screenshot.png' height='300' width='400'/><br>
🟢 Test Started: verifySessionHandling (🔢 Test Case ID: TC-021)

🌍 Test Environment:
🖥️ OS: Windows 11 10.0
☕ Java Version: 23.0.1
🌐 Browser: chrome 134.0.6998.118

✅ Test PASSED: verifySessionHandling (Duration: 17272ms)
<br><img src='E:/Folder/Galal/Courses/Digital Egypt Pioneers Initiative (DEPI) - Software Testing/DEPI Software Tester Graduation Project/automation-framework/reports/screenshots\TC-021\TC-021_screenshot.png' height='300' width='400'/><br>]]>
  </system-out>
</testsuite> <!-- tests.ui.SecurityTest -->
